test/tasks/StateOverrideManager.t.sol

```diff
@@ -246,53 +262,139 @@ contract StateOverrideManagerUnitTest is Test {
+            "ParentDefaultOverride: Must contain a nonce override"
+        );
+        assertEq(
+            parentDefaultOverride.overrides[1].value,
+            bytes32(expectedParentNonce),
+            "ParentDefaultOverride: Nonce override must match expected value"
+        );
+
+        // If child multisig is not set, we don't need to assert the child overrides.
+        if (childMultisig != address(0)) {
+            assertDefaultChildStateOverrides(allOverrides_, childMultisig, childMultisigNonce);
+        }
+    }
+
+    function assertDefaultChildStateOverrides(
+        Simulation.StateOverride[] memory allOverrides,
+        address childMultisig,
+        uint256 childMultisigNonce
+    ) internal pure {
+        assertTrue(
+            allOverrides.length >= 2,
+            "ChildDefaultOverride: Must be at least 2 overrides (parent default + child default)"
+        );
+        Simulation.StateOverride memory childDefaultOverride = allOverrides[1];
+
+        assertEq(
+            childDefaultOverride.contractAddress,
+            childMultisig,
+            "ChildDefaultOverride: Contract address must be the child multisig"
+        );
+        assertTrue(
+            childDefaultOverride.overrides.length == 5,
+            string.concat(
+                "ChildDefaultOverride: Default override must have 5 overrides, found: ",
+                LibString.toString(childDefaultOverride.overrides.length)
+            )
+        );
+        assertEq(
+            childDefaultOverride.overrides[0].key,
+            bytes32(uint256(0x4)),
+            "ChildDefaultOverride: Must contain a threshold override"
+        );
+        assertEq(
+            childDefaultOverride.overrides[0].value,
+            bytes32(uint256(0x1)),
+            "ChildDefaultOverride: Threshold override must be 1"
+        );
+        assertEq(
+            childDefaultOverride.overrides[1].key,
+            bytes32(uint256(0x5)),
+            "ChildDefaultOverride: Must contain a nonce override"
+        );
+        assertEq(
+            childDefaultOverride.overrides[1].value,
+            bytes32(childMultisigNonce),
+            "ChildDefaultOverride: Nonce override must match expected value"
+        );
+        assertEq(
+            childDefaultOverride.overrides[2].key,
+            bytes32(uint256(0x3)),
+            "ChildDefaultOverride: Must contain an owner count override"
+        );
+        assertEq(
+            childDefaultOverride.overrides[2].value,
+            bytes32(uint256(0x1)),
+            "ChildDefaultOverride: Owner count override must be 1"
+        );
+
         // Verify owner mapping overrides
+        // cast keccak 0x$(cast --to-uint256 1 | cut -c3-)$(cast --to-uint256 2 | cut -c3-)
         assertEq(
-            defaultOverride.overrides[3].key,
+            childDefaultOverride.overrides[3].key,
             bytes32(uint256(0xe90b7bceb6e7df5418fb78d8ee546e97c83a08bbccc01a0644d599ccd2a7c2e0)),
```

Comment:
Is this a slot calculation? Can we just do the computation inline here instead of hardcoding it with a cast command reference? Same for the hardcoded value below

Reply:
Sure, I was following what `Simulation.sol` was doing here: https://github.com/base/contracts/blob/7cca28be0f94134f9de1c6f409779b57d5f269b7/script/universal/Simulation.sol#L109

Reply:
Implemented here: https://github.com/ethereum-optimism/superchain-ops/pull/771/commits/2979b3c74d89bba4ca8fb0c6774bb122400ecad3